### PR TITLE
Lpscrypt/work/figma0.5 metrics

### PR DESCRIPTION
**Pull request by LPSCRYPT**
metrics & sidebar (sidebar icons to be added later)

### CODE REVIEW

File: src/components/topbar/topbar-global.tsx [hunk 1]
```diff
@@ -120,7 +120,7 @@ const TopbarGlobal: FC<TopbarProps> = ({ domain: _domain }) => {
         <div className="topHalf">
           <div className="topLeft">
             <Link to={'/'} className="network">
-              0://
+              0:/ <span style={{ marginLeft: '-5px' }}>/</span>
```
Comment (benefacto) [inline src/components/topbar/topbar-global.tsx:123]: Would it be possible to avoid using inline styles?

File: src/components/table/adbar.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import '../css/adbar.scss';
+import adbg from '../css/img/adbg.png';
+import wilderavatar from '../css/img/wilderavatar.png';
+
+interface AdBarProps {
+  domain: string;
+}
+
+const AdBar: FC<AdBarProps> = ({ domain: _domain }) => {
+  const context = useWeb3React<Web3Provider>();
```
Comment (benefacto) [inline src/components/table/adbar.tsx:18]: Is copying all this context boilerplate necessary right now?

File: src/components/css/subdomains.scss [hunk 3]
```diff
@@ -104,44 +111,100 @@ $page-margin: 180px;
         margin-top: 5px;
         margin-bottom: 5px;
         margin-right: 30px;
+
         position: relative;
         float: left;
-        width: 366px;
-        height: 178px;
+        width: 295px;
```
Comment (benefacto) [inline src/components/css/subdomains.scss:117]: Do we need to use static pixels heights on this element? Normally that could limit its responsiveness. If so, would it be possible to use max/min-width/height instead?

File: src/components/css/sidebar.scss [hunk 4]
```diff
@@ -4,18 +4,68 @@
...
+        rgba(40, 0, 68, 0.5) 0%,
+        rgba(62, 0, 105, 0.5) 100%
+      );
+      border-radius: 37px;
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+      .sideItem {
+        margin-left: 8px;
+        height: 50px;
```
Comment (benefacto) [inline src/components/css/sidebar.scss:51]: Max/min-width/height instead?

File: src/components/css/sidebar.scss [hunk 5]
```diff
@@ -4,18 +4,68 @@
...
+      img {
+        // border: 1px red solid;
+      }
+    }
+    .middleNav {
+      position: relative;
+      align-self: center;
+      margin: auto;
+      // top: 40vh;
+      height: 302px;
```
Comment (benefacto) [inline src/components/css/sidebar.scss:38]: Max/min-width/height instead?

File: src/components/css/sidebar.scss [hunk 6]
```diff
@@ -4,18 +4,68 @@
...
 
   position: fixed;
   top: 0;
   left: 0;
 
+  .sidebarContainer {
+    .topLogo {
+      position: absolute;
+      top: 24px;
+      height: 56px;
```
Comment (benefacto) [inline src/components/css/sidebar.scss:23]: Max/min-width/height instead?

File: src/components/css/adbar.scss [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+        }
+        .n3 {
+          margin-left: 8px;
+          background: #002f5b;
+          // background: red;
+          border-radius: 50%;
+          font-weight: normal;
+          // font-size: 16px;
+          width: 18px !important;
+          height: 18px;
```
Comment (benefacto) [inline src/components/css/adbar.scss:70]: Max/min-width/height instead?

File: src/components/css/adbar.scss [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+    margin-left: 20px;
+    position: absolute;
+    // border: 1px red solid;
+    color: #fff;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    .profileIcon {
+      img {
+        height: 64px;
```
Comment (benefacto) [inline src/components/css/adbar.scss:41]: Max/min-width/height instead?

**Review by benefacto — CHANGES_REQUESTED** [review]
This is really cool, @LPSCRYPT ! The metrics really looks sick and the site is really coming together.

I couple things need to happen before this pull request is ready to merge:
1. The latest changes from develop need to be merged into this branch and the conflicts resolved. Currently this branch is 6 commits behind develop.
2. Feel free to make changes based on my comments or mark them as resolved if you think the changes are unnecessary or you plan on making them later.
3. The overall responsiveness seems to be lacking a bit and I bet our client will bring this up. Essentially, the background for the bar does not spread across my entire screen. Feel free to resolve in the pull request or handle it later.

Anyways, the precision of your styling work is very impressive!

File: src/components/topbar/topbar-global.tsx (hunk 1 above)
Comment (LPSCRYPT) [reply src/components/topbar/topbar-global.tsx]: thanks for the feedback!
-fixed adbar background scaling
-took out inline styles
-merged with develop

**Review by benefacto — APPROVED** [review]

**Pull request merged**